Packs/PAN-OS/ReleaseNotes/2_6_7.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### PANOSQueryLogs
+
+Documentation and metadata improvements.
```

Comment:
```suggestion
Updated the documentation and metadata.
```

Changed this according to best practices mentioned here: https://xsoar.pan.dev/docs/documentation/release-notes